frontend/src/utils/questions.tsx

```diff
@@ -0,0 +1,50 @@
+import { type ColumnDef, type ColumnHelper } from '@tanstack/react-table';
+import { type QuestionData } from '../types/questions/questions';
+import { Stack, Tag } from '@chakra-ui/react';
+import QuestionComplexityTag from '../components/questions/QuestionComplexityTag';
+import QuestionViewIconButton from '../components/questions/QuestionViewIconButton';
+import React from 'react';
+
+export interface QuestionDataRowData extends QuestionData {
+  action?: undefined;
+}
+
+export const QuestionsTableColumns = (
+  columnHelper: ColumnHelper<QuestionDataRowData>,
+): Array<ColumnDef<QuestionDataRowData>> => {
+  return [
+    columnHelper.accessor('questionID', {
+      cell: (id): number => id.getValue(),
+      enableSorting: true,
+      header: 'ID',
+    }),
+    columnHelper.accessor('title', {
+      cell: (title): string => title.getValue(),
+      enableSorting: true,
+      header: 'Title',
+    }),
+    columnHelper.accessor('categories', {
+      header: 'Categories',
+      enableSorting: true,
+      cell: (categories) => (
+        <Stack direction="row">
+          {categories.getValue().map((category) => (
+            <Tag key={category}>{category}</Tag>
+          ))}
+        </Stack>
+      ),
+    }),
```

Comment:
```
columnHelper.accessor('categories', {
      header: 'Categories',
      enableSorting: true,
      cell: (categories) => (
        <Stack direction="row" spacing={4}>
          <Wrap>
            {categories.getValue().map((category) => (
              <WrapItem key={category}>
                <Tag>{category}</Tag>
              </WrapItem>
            ))}
          </Wrap>
        </Stack>
      ),
    }),```

Reply:
<img width="661" alt="Screenshot 2023-09-10 at 10 01 11 PM" src="https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g17/assets/96589109/bf8f64b5-d6ce-4d35-a610-8ea6fcc4e42f">

Reply:
Thanks for this! Was trying to implement it but got stuck : " )